src/xmipp/libraries/reconstruction/metadata_utilities.cpp

```diff
@@ -388,6 +388,12 @@ class ProgMetadataUtilities: public XmippProgram
 
     void doQuery()
     {
+    	if (mdIn.size()==0)
+    	{
+    		md2.clear();
+    		return;
+    	}
+
```

Comment:
Is it this on proposal?

Reply:
Yes, it protects some pathological cases in which the input metadata is empty. This should not normally happen, but sometimes, in automatic protocols, it happens.